guides/source/working_with_javascript_in_rails.md

```diff
@@ -318,6 +318,17 @@ added to the form that the `button_to` helper renders internally:
 <%= button_to "Delete post", post, method: :delete, form: { data: { turbo_confirm: "Are you sure?" } } %>
```

Comment:
Weird thing is that the following does work, it adds the confirm to the button instead of the form:
```erb
<%= button_to "Delete post", post, method: :delete, data: { turbo_confirm: "Are you sure?" } %>
```